src/services/cpd-service.js

```diff
@@ -107,11 +107,25 @@ export class CpdService {
 		return this.getRequest(api_path);
 	}
 
-	static getProgress(userId) {
-		if (!userId) {
-			return this.getRequest(CpdRoutes.RelativePath(CpdRoutes.Progress));
+	static getProgress(filters, userId) {
```

Comment:
as written, this will break current behaviour... I think you want 
```suggestion
	static getProgress(userId, filters) {
```